v22.1/spatial-indexes.md

```diff
@@ -50,25 +48,25 @@ CockroachDB uses the "divide the space" approach for the following reasons:
 - It requires no balancing operations, unlike [R-tree indexes](https://en.wikipedia.org/wiki/R-tree).
 - Inserts under this approach require no locking.
 - Bulk ingest is simpler to implement than under other approaches.
-- It allows advanced users to make a per-object tradeoff between index size and false positives during index creation. (See [Tuning spatial indexes](#tuning-spatial-indexes) below.)
```

Comment:
Not a blocker, but would it make sense to add a Vale linter that warns about these positional prepositions? If so, I can take that as a to-do.

Reply:
I think below, above etc aren't always positional, so I think a linter rule would throw too many false +.  I'm hoping that education writing for accessibility will help.